src/components/MapView/MapView.tsx

```diff
@@ -0,0 +1,94 @@
+import {View} from 'react-native';
+import {useFocusEffect} from '@react-navigation/native';
+import Mapbox, {MapState, MarkerView, setAccessToken} from '@rnmapbox/maps';
+import {forwardRef, memo, useCallback, useEffect, useImperativeHandle, useRef, useState} from 'react';
+
+import utils from './utils';
+import Direction from './Direction';
+import CONST from '../../CONST';
+
+import {MapViewProps, MapViewHandle} from './MapViewTypes';
+
+const MapView = forwardRef<MapViewHandle, MapViewProps>(({accessToken, style, mapPadding, styleURL, pitchEnabled, initialState, waypoints, directionCoordinates}, ref) => {
```

Comment:
NAB: Can we also specify here that the params are MapViewProps and the ref is a MapViewHandle, or is that already inferred?

Reply:
Also I think we want this component defined with the function keyword.